test/FranchiserFactory.invariant.t.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {Test, console2} from "forge-std/Test.sol";
+import {IVotingToken} from "src/interfaces/IVotingToken.sol";
+import {FranchiserFactory} from "src/FranchiserFactory.sol";
+import {Franchiser} from "src/Franchiser.sol";
+import {FranchiserFactoryHandler} from "test/handlers/FranchiserFactoryHandler.sol";
+import {VotingTokenConcrete} from "./VotingTokenConcrete.sol";
+
+contract FranchiseFactoryInvariantTest is Test {
+    FranchiserFactory factory;
+    FranchiserFactoryHandler handler;
+    VotingTokenConcrete token;
+
+    function setUp() public virtual {
+        token = new VotingTokenConcrete();
+        factory = new FranchiserFactory(IVotingToken(address(token)));
+        handler = new FranchiserFactoryHandler(factory);
+        bytes4[] memory selectors = new bytes4[](2);
+        selectors[0] = FranchiserFactoryHandler.handler_fund.selector;
+        selectors[1] = FranchiserFactoryHandler.handler_recall.selector;
+        targetSelector(FuzzSelector({addr: address(handler), selectors: selectors}));
+        targetContract(address(handler));
+    }
+
+    function invariant_fund_franchiser_code_size_non_zero() external view {
```

Comment:
Should we use invariant_ and then pascal case. @wildmolasses what are best practices?

Reply:
yeah, we did want to some day use this_type_of_casing for readability, but it's not PRed into scopelint yet, so I agree w/ alex.

Reply:
Switched to invariant_ followed by name with first word capitalized, as is done in UniStaker invariant tests. https://github.com/uniswapfoundation/franchiser/pull/1/commits/53c752c7c59a8eec81922bd3386444da0adcf2cf